framework/build.gradle

```diff
@@ -29,6 +35,19 @@ dependencies {
     testImplementation group: 'junit', name: 'junit', version: '4.13'
     testImplementation project(':framework-test')
     testImplementation sourceSets.testannotations.output
+
+    // AutoValue support in Returns Receiver Checker
+    testImplementation "com.google.auto.value:auto-value-annotations:${versions.autoValue}"
+    testImplementation "com.google.auto.value:auto-value:${versions.autoValue}"
+
+    // Lombok support in Returns Receiver Checker
+    testImplementation "org.projectlombok:lombok:${versions.lombok}"
+}
+
+configurations{
+    testImplementation{
+        canBeResolved = true
```

Comment:
@nimakarimipour can we add a comment here indicating why this `canBeResolved` flag is needed?